app/rne-theme/colors.ts

```diff
@@ -39,4 +41,45 @@ const colors = {
   },
 }
 
-export default colors
+const dark = {
+  primary: "#1627C4",
+  primary3: "#1627C4",
+  primary5: "#4453E2",
+  primary6: "#6775F3",
+  primary7: "#A4ABF2",
+  primary8: "#CACFFD",
+  primary9: "#E6E8FA",
+  primary10: "#F0F0F7",
+  secondary: "#FB5607",
+  secondary5: "#FB5607",
+  secondary8: "#FFD0BA",
+  success: "#00A700",
+  success5: "#00A700",
+  success9: "#C6EBC6",
+  error: "#DC2626",
+  error4: "#DC2626",
+  error5: "#EF4444",
+  error8: "#FECACA",
+  error9: "#FEE2E2",
+  warning: "#F59E0B",
+  warning4: "#F59E0B",
+  warning9: "#FEF3C7",
+  white: "#000000", // <-- this is the only change
+  black: "#FFFFFF", // <-- this is the only change
+  grey5: "#61637A",
+  grey7: "#9292A0",
+  grey8: palette.lightGrey, // <-- this is the only change
```

Comment:
Are all other colors okay to stay the same in dark mode? Both the new currency input and Theo's home screen components use this new color theme. We will be blocked on merging our PRs until dark mode is supported, and that will require having the dark theme completed.

Reply:
I'm settings the darkMode colors as I use them right now. 

it helps me to understand how they translate from light to dark. it's not always a 1-1 mapping in my design to darkMode (not ideal but we'll have to think about it more later)

Reply:
maybe I'm mistaken what you're saying, but the current component are quite inconsistent already (ie: some background screen are sometimes `white`, sometime `lighterGrey`, and so the rest of the button are not consistent at all. there is quite a bit of refactoring work to do on the styles but I think this is part of the redesign to bring more consistency). I don't think we should be wary to "break" existing design, we can fix it as we go as we're refactoring other screen to introduce the new design.

Reply:
I believe the current components don't use this theme at all, but the new redesigned components do. That's why I would keep this theme in a good state so that it works for the new components. For the old components, I wouldn't use the the theme object.

Reply:
> That's why I would keep this theme in a good state so that it works for the new components

I don't think the new theme is very understandable as of now because of how the type is been done.

here is an example of all the typed showing up:

![image](https://user-images.githubusercontent.com/11156370/228054567-5daafc8e-dbc9-48b5-8407-21c88a4c058e.png)

yet, they're really mixed

<img width="304" alt="image" src="https://user-images.githubusercontent.com/11156370/228054786-09989773-803f-4ff3-a51c-f7014a2a811a.png">